### DISCUSSION TITLE
Preview the publication before going live

### DISCUSSION THREAD

**Discussion by gabivechiatto** [directus/directus#11921] [Feature Requests]
There could be the possibility that, while we are putting together the publication, we have the option of previewing it before publishing or scheduling the publication. 

It can be in the same window where we are editing the publication, or in a specific section for the preview

Thank you so much!

**Comment by azrikahar** [comment]
Definitely useful! There are some discussions on this topic such as #5897 and #11301. Hope they are helpful 👍

**Reply by gabivechiatto** [reply]
Nice! Thank you so much!

**Comment by tedsteen** [comment]
The developer experience for this feature in storyblok is pretty nice.
https://www.storyblok.com/tp/add-a-headless-cms-to-svelte-in-5-minutes

Posting for inspiration

**Comment by jonlambert** [comment]
Linking this conversation as I believe these are related features: 
https://github.com/directus/directus/discussions/2975

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
